tfbpshiny/misc/dto_distributions_module.py

```diff
@@ -4,7 +4,7 @@
 from shiny import Inputs, Outputs, Session, module, reactive
 from shinywidgets import output_widget, render_plotly
 
-from ..misc.generic_distribution_plot import create_distribution_plot
+from ..utils.create_distribution_plot import create_distribution_plot
```

Comment:
all of the directories other than `utils` contain shiny objects. Utils is for general functions etc.